### PR TITLE
Add Pull Request template

### PR DESCRIPTION
**Pull request by adrienball**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+**Description**:
+- ...
+
+**Checklist**:
+- [ ] My PR is ready for code review
+- [ ] I have added some tests, if applicable
```
Comment (ClemDoum) [inline .github/PULL_REQUEST_TEMPLATE.md]: I would add, '"and run the whole test suite"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+- [ ] My code respects the [PEP8 style guide](https://www.python.org/dev/peps/pep-0008/)
```
Comment (ClemDoum) [inline .github/PULL_REQUEST_TEMPLATE.md]: I don't know if that is necessary. I think that for now we have no guaranty that our current code base is PEP8 compliant. The only thing we know is that the codebase is compliant with the Google-like linting rules that we have in the `pylintrc`

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ClemDoum) [reply .github/PULL_REQUEST_TEMPLATE.md]: I would also add "even linting tests, this way we can get rid of the next point"

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (ClemDoum) [reply .github/PULL_REQUEST_TEMPLATE.md]: + I find it very hard to know if your code is PEP8 compliant from juste the guidelines provided in the URL. The linting test has the advantage to tell you precisely what's wrong

Comment (adrienball) [reply .github/PULL_REQUEST_TEMPLATE.md]: Indeed

**Pull request merged**